ROMFS/px4fmu_common/init.d/rc.fw_defaults

```diff
@@ -14,7 +14,7 @@ param set-default MAV_TYPE 1
 # Default parameters for fixed wing UAVs.
 #
 param set-default COM_POS_FS_DELAY 5
-param set-default COM_POS_FS_EPH 15
+param set-default COM_POS_FS_EPH 50
```

Comment:
Handling this contextually is something we should think about for VTOL at some point.

Reply:
Probably. Though they're anyway not the trigger currently when losing GPS in hover. I would need to check which condition triggers it, but every VTOL (at least without optical flow) immediately goes to position loss failsafe on losing GPS in hover.